cinema_service/settings.py

```diff
@@ -27,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]
```

Comment:
The `ALLOWED_HOSTS` setting is set to `['*']`, which allows all hosts. This should be restricted to specific domains in a production environment for security reasons.